ts/features/fims/components/FimsSuccessBody.tsx

```diff
@@ -0,0 +1,231 @@
+          {I18n.t("FIMS.consentsScreen.bottomSheet.body")}
+        </Label>
+        <VSpacer size={8} />
+        <Label weight="Regular">
+          {I18n.t("FIMS.consentsScreen.bottomSheet.body2")}
+          <H6
+            onPress={() => privacyUrl && openWebUrl(privacyUrl)}
+            color="blueIO-500"
+          >
+            {I18n.t("FIMS.consentsScreen.bottomSheet.bodyPrivacy")}
+          </H6>
+        </Label>
+      </>
+    ),
+    snapPoint: [340]
+  });
+
+  return (
+    <>
+      <ForceScrollDownView
+        style={IOStyles.horizontalContentPadding}
+        contentContainerStyle={{
+          flex: 1,
+          justifyContent: "space-between"
+        }}
+      >
+        <View>
+          <VSpacer size={24} />
+          <View style={{ flexDirection: "row", alignItems: "center" }}>
+            <View style={styles.outlineContainer}>
+              <Icon name="productIOApp" size={30} color="blueIO-500" />
+            </View>
+            <HSpacer size={8} />
+            <Icon name="transactions" color="grey-450" />
+            <HSpacer size={8} />
+            <Avatar logoUri={serviceLogo} size={"small"} />
+          </View>
+
+          <VSpacer size={24} />
+          <H2>{I18n.t("FIMS.consentsScreen.title")}</H2>
+          <Subtitle />
+
+          <VSpacer size={24} />
+          <ButtonText
+            weight="Bold"
+            color="blueIO-500"
+            onPress={BottomSheet.present}
+          >
+            {I18n.t("global.why")}
+          </ButtonText>
+          <VSpacer size={24} />
+          <ListItemHeader label="Dati richiesti" iconName="security" />
+          <ClaimsList claims={consents.claims} />
+          <VSpacer size={24} />
+
+          <PrivacyInfo privacyUrl={privacyUrl} />
+
+          <VSpacer size={32} />
+        </View>
+        <SafeAreaView>
+          <BlockButtons
+            type="TwoButtonsInlineHalf"
+            primary={{
+              type: "Outline",
+              buttonProps: {
+                label: I18n.t("global.buttons.cancel"),
+                onPress: () => {
+                  // BE call consents._links.cancel
+                  navigation.goBack();
+                }
+              }
+            }}
+            secondary={{
+              type: "Solid",
+              buttonProps: {
+                label: I18n.t("global.buttons.consent"),
+                icon: "security",
+                iconPosition: "end",
+                onPress: () =>
+                  dispatch(
+                    fimsGetRedirectUrlAndOpenIABAction.request(
+                      // eslint-disable-next-line no-underscore-dangle
+                      { acceptUrl: consents._links.confirm.href }
+                    )
+                  )
+              }
+            }}
+          />
+        </SafeAreaView>
+      </ForceScrollDownView>
+      {BottomSheet.bottomSheet}
+    </>
+  );
+};
+
+const ClaimsList = ({ claims }: ClaimsListProps) => (
```

Comment:
If it is a component, let's put it on its own file (including its properties), otherwise let's transform it to a function

Reply:
As `resolved` above, let's leave it as it is